src/pymmcore_widgets/_mda_widget/_mda_widget.py

```diff
@@ -537,7 +537,7 @@ def set_state(self, state: dict | MDASequence | str | Path) -> None:
         else:
             self.stage_pos_groupBox.setChecked(False)
 
-    def _get_state(self) -> MDASequence:
+    def get_state(self) -> MDASequence:
```

Comment:
curious why you made this private.  this would appear to be the one main "public" kinda thing from this widget.  It's essentially the main way for a user to get the MDASequence out right?  Are you replacing it with something else?

Reply:
me=dumb... this is *making* it public